src/server/scripts/Kalimdor/zone_azuremyst_isle.cpp

```diff
@@ -716,6 +717,23 @@ class npc_death_ravager : public CreatureScript
     }
 };
 
+// 29528 -  Inoculate Nestlewood Owlkin
+class spell_inoculate_nestlewood : public AuraScript
+{
+    PrepareAuraScript(spell_inoculate_nestlewood);
+
+    void PeriodicTick(AuraEffect const* /*aurEff*/)
+    {
+        if (GetTarget()->GetTypeId() != TYPEID_UNIT) // prevent error reports in case ignored player target
```

Comment:
Needed? spell 29528 effect 0 uses `Targets (38, 0) (TARGET_UNIT_NEARBY_ENTRY, NO_TARGET)`,
TARGET_UNIT_NEARBY_ENTRY should only result in TARGET_OBJECT_TYPE_UNIT ?

Reply:
well we will keep it for now, for the sake of sync with 3.3.5